recipe/meta.yaml

```diff
@@ -12,10 +12,10 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 4
   skip: true  # [win and vc<14]
   run_exports:
-    - pin_compatible('quantlib', max_pin='x')
+    - quantlib
```

Comment:
```suggestion
    - {{ pin_compatible('quantlib', max_pin='x') }}
```

Reply:
This doesn't appear to be valid, see the quantlib python staged recipe breaking

Reply:
Ah you had forgotten to put it in {{